src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  counter-reset: day-number;
+  flex-wrap: wrap;
+  display: flex;
+  width: $calendar-width;
+  gap: $calendar-day-gap;
+
+  .calendar__day {
+    background-color: #eee;
+    border: 1px solid #000;
+    width: $calendar-day-size;
+    height: $calendar-day-size;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition:
+      transform $transition-duration,
+      background-color 0.5s;
```

Comment:
The transition duration should be stored in a variable, especially if it's used in multiple places. This will make it easier to change the transition duration in the future if needed.